sqlparse/keywords.py

```diff
@@ -959,14 +959,3 @@
 KEYWORDS_MSACCESS = {
     'DISTINCTROW': tokens.Keyword,
 }
-
-KEYWORDS_DUCKDB = {
-    # duckdb supports FROM-first syntax which may even omit the SELECT, so
-    # e.g. `from dataframe` is a full, valid query
-    'FROM': tokens.Keyword.DML,
-
-    # https://duckdb.org/docs/sql/statements/pivot
-    'PIVOT': tokens.Keyword.DML,
-    'PIVOT_WIDER': tokens.Keyword.DML,
-    'UNPIVOT': tokens.Keyword.DML,
-}
```

Comment:
These were having no effect because my adding them to the SQL regex was taking precedence and in the regex they were mapped to `Keyword` rather than `Keyword.DML`. But I think it would be too invasive to change the type of the `FROM` keyword, and the only reason I was mapping these to `DML` was so that `Statement.get_type()` would return the corresponding keyword instead of `UNKNOWN`, but I think it's better to handle that differently - see below.